tests/test_project.py

```diff
@@ -424,6 +424,108 @@ def test_update_some_with_imports(repos_tmpdir):
     manifest = Manifest.from_file(topdir=ws)
     assert manifest.get_projects(['Kconfiglib'])[0].is_cloned()
 
+def test_update_submodules(repos_tmpdir):
```

Comment:
nit: consider adding some negative test case to check that a submodule which is not in the manifest is not cloned